plugins.json

```diff
@@ -144,6 +144,14 @@
       "author":"Kyle Fuller, Boris Bügling",
       "url":"https://github.com/CocoaPods/cocoapods-packager",
       "description":"Generate a framework or static library from a podspec."
+    },
+    {
+      "gem":"cocoapods-links",
+      "name":"Pod links",
```

Comment:
I think `Foo` is preferred when the tool functions standalone, and `CocoaPods Foo` otherwise, unless there's a compelling reason for something else.